pkg/cmd/purge/purge.go

```diff
@@ -0,0 +1,100 @@
+package purge
+
+import (
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/purge"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	urls      string
+	wildcard  string
+	cachekeys string
+	Layer     string
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azion purge --wildcard "www.example.com/*"
+        $ azion purge --urls "www.example.com,www.pudim.com"
+        $ azion purge --cache-key "www.domain.com/@@cookie_name=cookie_value,www.domain.com/test.js"
+        `),
+		RunE: func(cmd *cobra.Command, _ []string) error {
```

Comment:
externalize the func, to make the code cleaner

Reply:
Never seen anyone else do this, but I could 👍

Reply:
But that's a suggestion, feel free 😄